chat_exporter/chat_exporter.py

```diff
@@ -55,23 +55,26 @@ async def export(ctx):
         await ctx.channel.send(embed=transcript_embed, file=transcript_file)
 
 
-async def generate_transcript(channel: discord.TextChannel, tz_info="US/Eastern"):
+async def generate_transcript(channel: discord.TextChannel, tz_info="US/Eastern", messages: list = None):
     global eastern
     eastern = timezone(tz_info)
 
     # noinspection PyBroadException
     try:
-        transcript = await produce_transcript(channel)
+        if messages:
+            transcript = await produce_transcript(channel, messages)
+        else:
+            transcript = await produce_transcript(channel)
```

Comment:
This doesn't seem clean. 

If you were to use this method, surely you can just do:
`transcript = await produce_transcript(channel, messages)`

If messages wasn't passed originally, then `messages` would be `None`. Then when it reaches `product_transcript` it'll either be a value or None (where then it'll proceed to the `or`)

Reply:
Fixed